### PR TITLE
Add state-flow.state/runner to access the runner within flows

### PR DESCRIPTION
**Pull request by dchelimsky**

### CODE REVIEW

File: test/state_flow/state_test.clj [hunk 1]
```diff
@@ -2,6 +2,7 @@
   (:require [cats.core :as m]
             [cats.monad.exception :as e]
             [clojure.test :as t :refer [deftest is testing]]
+            [state-flow.core :as state-flowo]
```
Comment (rafaelzlisboa) [inline test/state_flow/state_test.clj:5]: typo

**Comment by gabrielgiussi** [comment]
Wich is the use case of this? When I could want to access the runner from a flow?

**Comment by dchelimsky** [comment]
> Which is the use case of this? When I could want to access the runner from a flow?

It's for helpers that need to run a flow within a flow in order to wrap it in some other function (e.g. `with-redefs`.

File: src/state_flow/state.clj [hunk 2]
```diff
@@ -113,3 +113,9 @@
   (if (state? value)
     value
     (return value)))
+
+(defn runner
```
Comment (sovelten) [inline src/state_flow/state.clj:117]: Now this is part of core, no? runner is defined in core and meta is also something related to core.

File: test/state_flow/state_test.clj [hunk 3]
```diff
@@ -2,6 +2,7 @@
   (:require [cats.core :as m]
             [cats.monad.exception :as e]
             [clojure.test :as t :refer [deftest is testing]]
+            [state-flow.core :as state-flow]
```
Comment (sovelten) [inline test/state_flow/state_test.clj:5]: You shouldn't declare core on state_test.clj namespace

**Pull request merged**